docs/strategy-callbacks.md

```diff
@@ -653,8 +655,7 @@ class DigDeeperStrategy(IStrategy):
             return None
 
         filled_buys = trade.select_filled_orders('buy')
-        count_of_buys = len(filled_buys)
-        
+        count_of_buys = trade.nr_of_successful_buys
```

Comment:
I found that these 2 lines are still necessary - as they're used as part of the conditions below (even though it's no longer used for the conditional). 
I've therefore re-added them.